demos/imagenet_classification_demo/grid_mat.hpp

```diff
@@ -0,0 +1,176 @@
+        int textPadding = 10;
+
+        cv::putText(outImg,
+                    cv::format("FPS: %0.01f", avgFPS),
+                    cv::Point(textPadding, textSize.height + textPadding),
+                    cv::FONT_HERSHEY_PLAIN, fontScale, textColor, thickness);
+        cv::putText(outImg,
+                    cv::format("Latency: %dms", static_cast<int>(avgLatency * 1000)),
+                    cv::Point(textPadding, (textSize.height + textPadding) * 2),
+                    cv::FONT_HERSHEY_PLAIN, fontScale, textColor, thickness);
+        
+        if (showAccuracy) {
+            std::string accuracyMessage = "Accuracy (top 0): 0.000";
+            cv::Size accuracyMessageSize = cv::getTextSize(accuracyMessage,
+                                                       cv::FONT_HERSHEY_PLAIN, fontScale, thickness, &baseline);
+            cv::putText(outImg,
+                        cv::format("Accuracy (top %d): %.3f", FLAGS_nt, accuracy),
+                        cv::Point(outImg.cols - accuracyMessageSize.width - textPadding, textSize.height + textPadding),
+                        cv::FONT_HERSHEY_PLAIN, fontScale, textColor, thickness);
+        }
+        
+        std::string testMessage = "Testing, please wait...";
+        cv::Size testMessageSize = cv::getTextSize(testMessage,
+                                                   cv::FONT_HERSHEY_PLAIN, fontScale, thickness, &baseline);
+        if (isFpsTest) {
+            cv::putText(outImg,
+                        testMessage,
+                        cv::Point(outImg.cols - testMessageSize.width - textPadding,
+                                  (textSize.height + textPadding) * 2),
+                        cv::FONT_HERSHEY_PLAIN, fontScale, cv::Scalar(50, 50, 255), thickness);
+        }
+    }
+
+    cv::Mat getMat(std::vector<long long> &endTimes, bool showLabels) {
+        while (!updateList.empty()) {
+            cv::Mat frame = std::get<0>(updateList.front());
+            std::string predictedLabel = std::get<1>(updateList.front());
+            cv::Scalar textColor = (std::get<2>(updateList.front())
+                                                ? cv::Scalar(75, 255, 75) : cv::Scalar(50, 50, 255));
+            updateList.pop_front();
+
+            if (updateList.empty()) {
+                if (prevImgs.size() >= FLAGS_b && points.size() > 1) {
+                    int prevSourceID = currSourceID - FLAGS_b;
+                    if (prevSourceID < 0) {
+                        prevSourceID += points.size();
+                    }
+                    cv::resize(prevImgs.front(),
+                               outImg(cv::Rect(points[prevSourceID], cellSize)),
+                               cellSize);
+                    prevImgs.pop();
+                }
+                
+                cv::resize(frame, frame, cellSize);
+
+                int thickness = cellSize.width / 20;
+                int baseline = 0;
```

Comment:
This var should be moved inside of the following `if` because it is used only there.